port/src/devcons.rs

```diff
@@ -20,12 +20,13 @@ pub trait Uart {
     fn putb(&self, b: u8);
 }
 
-static mut EARLY_CONS: Option<&'static mut dyn Uart> = None;
 static CONS: Lock<Option<&'static mut dyn Uart>> = Lock::new("cons", None);
 
-pub struct Console;
+/// LockingConsole is the what should be used in almost all cases, as it ensures
+/// threadsafe use of the console.
+pub struct LockingConsole;
 
-impl Console {
```

Comment:
I wouldn't rename this; `Console` will always be locking, the exception is for the panic handler.